corehq/apps/users/dbaccessors.py

```diff
@@ -163,7 +163,7 @@ def _get_users_by_filters(domain, user_type, user_filters, count_only=False):
         query = _get_es_query(domain, user_type, user_filters)
         if count_only:
             return query.count()
-        user_ids = query.scroll_ids()
+        user_ids = query.get_ids()
```

Comment:
worth noting here is that `get_ids` performs a single request to ES.  I agree that holding all IDs in memory is probably fine, but I'm less certain that fetching them all in a single request won't have issues (it might be fine, I really don't know).

An easy alternative is to still use a scroll query, but fully evaluate it up-front:
```suggestion
        user_ids = list(query.scroll_ids())
```

You could also do a limit/offset thing in chunks if the tradeoffs there are acceptable (iterating over a changing dataset) and you don't want the overhead of creating a scroll context.